tutorial/README.md

```diff
@@ -21,14 +21,14 @@ As you complete each task you'll see the message update with a green checkmark.
 1. A Slack team.
 Before anything else you'll need a Slack team. You can [Sign into an existing Slack workspace](https://get.slack.help/hc/en-us/articles/212681477-Sign-in-to-Slack) or you can [create a new Slack workspace](https://get.slack.help/hc/en-us/articles/206845317-Create-a-Slack-workspace) to test your app first.
 
-2. A terminal with Python 3.6+ installed.
+2. A terminal with Python 3.7+ installed.
 Check your installation by running the following command in your terminal:
 ```
 $ python3 --version
--> Python 3.6.7
+-> Python 3.7.0
```

Comment:
NIT: lets use 3.7.17 since its the latest [3.7 version](https://peps.python.org/pep-0537/) and a security release